File: app/frontend/ea-project/src/models/input/CreateClubInput.js
import BaseInput from "./BaseInput";

export default class CreateClubInput extends BaseInput {
  constructor(id = 0, name = '', isActive = false, creationDate = new Date(), lazyLoad = false) {
    super(lazyLoad);
    this.id = id;
    this.name = name;
    this.isActive = isActive;
    this.creationDate = creationDate;
  }
}

File: app/backend/src/main/java/eaproject/input/CreateClubInput.java
package eaproject.input;

import java.io.Serializable;
import java.sql.Timestamp;

public class CreateClubInput extends BaseInput implements Serializable {

    private String name;
    private boolean isActive;
    private Timestamp creationDate;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public boolean getIsActive() {
        return isActive;
    }

    public void setIsActive(boolean isActive) {
        this.isActive = isActive;
    }

    public Timestamp getCreationDate() {
        return creationDate;
    }

    public void setCreationDate(Timestamp creationDate) {
        this.creationDate = creationDate;
    }
}
